openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1499,9 +1499,14 @@ Access Tokens represent End-User authorization and consent to issue certain Cred
 
 To sender-constrain Access Tokens, see the recommendations in Section 4.10.1 in [@!I-D.ietf-oauth-security-topics]. If Bearer Access Tokens are stored by the Wallet, they MUST be stored in a secure manner, for example, encrypted using a key stored in a protected key store.
 
-# Privacy Considerations
+# Privacy Considerations {#privacy-considerations}
 
-TBD
+Privacy harms can occur if information about a person is released to another party without the person's consent.
+True consent involves both the person knowing what information is being released and knowing who it is being released to, and for what purpose.
```

Comment:
this sounds more relevant to presentation than issuance. it can be relevant for the issuance too, but if so, needs to be more precise.

Reply:
I guess that generically applies to all kinds of flows. In issuance that would mean, a credential is issued to a wallet (or an entity posing as a wallet) not under the control of the actual user.

Reply:
I incorporated Torsten's feedback.